src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -196,6 +261,39 @@ export default Vue.extend<Data, Methods, unknown, Props>({
                         "An error occurred fetching workflow details";
                 });
         },
+        getWorkflowMetadata(){
+            api.get(`/workflows/${this.selectedWorkflowKey}/`)
+                .then(({data}) => {
+                    this.workflowMetadata = workflowRunDetailsToMetadata(data.data)
+                    this.getReportDependencies()
+                })
+                .catch((error) => {
+                    this.error = error;
+                    this.defaultMessage =
+                        "An error occurred fetching workflow details";
+                });
+        },
```

Comment:
This repeats logic from the rerun method above. Functionally it's very distinct but could perhaps merge both methods with some conditional logic?